.changeset/curly-buses-allow.md

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
But without the changeset, how it will update the dependencies to use the correct packages?
When the changeset wasn't available, the `test-cli-with-database` was failing using old versions of dependencies

Reply:
Changesets does not really know anything about the actual versions used. It only looks at what version bumps are specified across all files in `.changesets/` and replaces the versions accordingly. 

With this change, we will have two changesets for all packages that both contain a `patch` bump. This won't really do anything except tell changeset a second time, that versions should be replaced with the next patch version in the range.

Reply:
What I mean is, how `test-cli-with-database` will get the correct version from the PRs to test the project if it is set to use the version "preview"?
My tests were all failing using old packages like getting values from types, until I added the changeset.
So, maybe the issue is somewhere else, and I got the impression that something was reading the changeset to invalidate some sort of cache, etc

Reply:
> test-cli-with-database

I guess we will have to change the way that action is set up and executed. Until now, it hasn't relied on published versions, but instead used `medusa-dev-cli`. 

Now, with the dev CLI removed, we will have to use an actually published version – unless we can set up linking (e.g. with resolutions) in the action too.